lib/puppet/puppet_x/helper.rb

```diff
@@ -74,10 +70,16 @@ def os_name(scope, platform_name)
     return 'Amazon Linux'
   end
 
-  rhel_base = ['RedHat', 'CentOS', 'OracleLinux', 'Scientific', 'Rocky', 'AlmaLinux']
+  if ['RedHat', 'Scientific', 'Rocky', 'AlmaLinux'].any? { |base| fact_os_name.casecmp(base).zero? }
+    return '*RHEL*'
+  end
+
+  if fact_os_name.casecmp('OracleLinux').zero?
+    return '*Oracle*'
+  end
 
-  if rhel_base.any? { |base| fact_os_name.casecmp(base).zero? }
-    return 'RHEL/CentOS/Oracle'
+  if fact_os_name.casecmp('CentOS').zero?
+    return '*CentOS*'
   end
```

Comment:
Should we just make this:
``` '*RHEL*' ```
since it's all in the RHEL family?

Reply:
So for OS i've seen the api have these values:

`RHEL`, `RHEL/CentOS`, `RHEL/CentOS/Oracle`

I know RHEL 9 is just `RHEL`, but I wasn't sure what path they were going to do in the future. IE: is there going to just be a `Oracle` if thats the case then just using `RHEL` wouldn't work.

Reply:
Updated to use just RHEL